simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/OpsWorksLarge.scala

```diff
@@ -13,7 +13,7 @@ import io.gatling.http.Predef._
 class OpsWorksLarge extends SimulationWithScenario {
 
 // 	val httpProtocol = http
-// 		.baseURL("https://aws-lrg-mom.us-west-2.compute.internal:8140")
+// 		.baseURL("https://aws-lrg-master.us-west-2.compute.internal:8140")
```

Comment:
Potentially dumb question, but... Are these URLs for existing servers that have been set up for OpsWorks testing or are they provisioned / assigned for each test run?

Reply:
:noidea: